util/index.ts

```diff
@@ -0,0 +1,107 @@
+import inquirer, { Answers } from 'inquirer';
+import inquirerFuzzyPath, { FuzzyPathQuestionOptions } from 'inquirer-fuzzy-path';
+
+import { Lang } from '../resources/languages';
+
+import { run as findMissingTranslations } from './find_missing_translations';
+import { default as generateEffectIds } from './gen_effect_id';
+import { run as translateTimeline } from './translate_timeline';
+
+declare module 'inquirer' {
+  interface QuestionMap<T> {
+    fuzzypath: FuzzyPathQuestionOptions<T>;
+  }
+}
+
+const dataFilesMap = {
+  'effect_id.ts': generateEffectIds,
+} as const;
+
+inquirer.registerPrompt('fuzzypath', inquirerFuzzyPath);
+
+const run = () => {
+  inquirer.prompt([{
+    type: 'list',
+    name: 'action',
+    message: 'What do you want to do?',
+    choices: [
+      'Generate common data files',
+      'Translate Raidboss timeline',
+      'Find Raidboss missing translations',
+    ],
+  }]).then((answer: Answers) => {
+    if (answer.action === 'Generate common data files')
```

Comment:
What about using choice objects, where instead of `'Generate common data files'` (and then repeating this entire English string, which may be updated), you use `{ name: 'Generate common data files', key: 'generate' }`.  I expect also these strings would get translated at some point too, so having them a little bit more abstract will help will that.